packages/flutter_tools/lib/src/web/bootstrap.dart

```diff
@@ -4,6 +4,320 @@
 
 import 'package:package_config/package_config.dart';
 
+const String _currentDirectoryScript = r"""
```

Comment:
@ditman could you take a look at the diff in this library? I've reviewed the other changes, but I don't know how any of the existing bootstrapping works anyway.